pcsx2/GS/GS.cpp

```diff
@@ -716,20 +715,14 @@ void GSgetStats(std::string& info)
 
 void GSgetTitleStats(std::string& info)
 {
-	const char* api_name = HostDisplay::RenderAPIToString(s_render_api);
-	const char* hw_sw_name = (GSConfig.Renderer == GSRendererType::Null) ? " Null" : (GSConfig.UseHardwareRenderer() ? " HW" : " SW");
-	const char* deinterlace_mode = theApp.m_gs_interlace[static_cast<int>(GSConfig.InterlaceMode)].name.c_str();
-
-#ifndef PCSX2_CORE
 	int iwidth, iheight;
 	GSgetInternalResolution(&iwidth, &iheight);
 
-	info = StringUtil::StdStringFromFormat("%s%s | %s | %dx%d", api_name, hw_sw_name, deinterlace_mode,  iwidth, iheight);
```

Comment:
Looks like a merge conflict gone wrong here (wx and qt should emit different info).

Reply:
Yeah didn't know what I did, probably didn't pay attention I did see the issue but not the cause.